src/engraving/dom/measure.cpp

```diff
@@ -3433,6 +3451,13 @@ void Measure::triggerLayout() const
     }
 }
 
+void Measure::triggerLayout(staff_idx_t staffIdx) const
+{
+    if (prev() || next()) { // avoid triggering layout before getting added to a score
```

Comment:
This looks a bit hacky, what problems was it causing? Looks like my previous comment about triggerLayout may help, cause if it's called out of the setter it can be better managed. This also has an edge case on a score with a single measure.

Reply:
> what problems was it causing? 

I have no idea actually, but I copied it from the normal `triggerLayout`. But it should indeed be possible to remove it, good point. (I think I'm not going to touch the existing `triggerLayout` in this PR, though.)